src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -125,7 +126,11 @@ void apply_remap(Subgraph& fcall, const ClosureRemap& m) {
     // empty tensors by default.
     for (auto&& i : m.closure_remap) {
         new_lazy_closure.push_back(fcall._lazy_closure[i]);
-        new_closure.push_back(fcall._closure[i]);
+        if (fcall._closure[i]) {
```

Comment:
Tricky stuff. Without this check we might push into closure an empty lazy_closure. After that we would get both empty. Now we might already have _closure -> use it. I believe it was the case for host gather tensor